pulsar-broker/src/main/java/org/apache/pulsar/broker/resourcegroup/ResourceGroupService.java

```diff
@@ -117,19 +122,24 @@ public ResourceGroup resourceGroupGet(String resourceGroupName) {
      *
      * @throws if RG with that name does not exist.
      */
-    public void resourceGroupUpdate(ResourceGroupConfigInfo rgConfig) throws PulsarAdminException {
+    public void resourceGroupUpdate(String rgName, org.apache.pulsar.common.policies.data.ResourceGroup rgConfig)
+      throws PulsarAdminException {
         try {
             checkNotNull(rgConfig);
         } catch (NullPointerException e) {
             throw new IllegalArgumentException("ResourceGroupUpdate: Invalid null ResourceGroupConfigInfo");
```

Comment:
Nit: `ResourceGroupConfigInfo` should be `ResourceGroup` now.

Reply:
if it is ok with you, I can take care of it it in my next commit.

Reply:
done.